### PR TITLE
adding spring properties

### PR DESCRIPTION
**Pull request by Keyshal-211**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?

Please describe the test strategy you followed. If not tested, please explain why.

- [ ] automated unit test
- [ ] automated integration test
- [ ] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.
    
## Checklist before requesting a review

- [X] My code is easy to understand
- [X] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [X] My code fulfills all acceptance criteria
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [ ] I have made corresponding changes to the documentation or
  the [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I made no breaking changes in the database schema or if so, I will perform a database migration

## Checklist for reviewer

- The code is easy to understand
- The code follows
  the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of
  this project
- The code is tested or if not, the reason is documented or discussed
- The added and existing tests reasonably cover the code change
- The code has no breaking changes in the database schema or if so, the assignee is aware of it and
  will perform a database migration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,36 @@
...
+| spring.jpa.show-sql                       | Show SQL queries in logs                  | true                                             | true                                                                 |
+| spring.sql.init.continue-on-error         | Continue on SQL init error                | true                                             | true                                                                 |
+| spring.jpa.hibernate.ddl-auto             | Hibernate DDL auto strategy               | create                                           | update                                                               |
+| hibernate.create_empty_composites.enabled | Enable empty composite types in Hibernate | true                                             | true                                                                 |
+| DAPR_GRPC_PORT                            | Dapr gRPC Port                            | 50001                                            | 50001                                                                |
```
Comment (PaulBredl) [inline README.md]: This is not set in DEV

File: README.md [hunk 2]
```diff
@@ -4,15 +4,36 @@
...
+| DAPR_HTTP_PORT                            | Dapr HTTP Port                            | 3500                                             | 3500                                                                 |
```
Comment (PaulBredl) [inline README.md]: This is wrong, it's 4000

File: README.md [hunk 3]
```diff
@@ -4,15 +4,36 @@
...
+| spring.profiles.active                    | Active Spring profile                     | dev                                              | prod                                                                 |
+| spring.jpa.properties.hibernate.dialect   | Hibernate dialect for PostgreSQL          | org.hibernate.dialect.PostgreSQLDialect          | org.hibernate.dialect.PostgreSQLDialect                              |
+| spring.datasource.url                     | PostgreSQL database URL                   | jdbc:postgresql://localhost:4032/content-service | jdbc:postgresql://content-service-db-postgresql:5432/content-service |
+| spring.datasource.username                | Database username                         | root                                             | gits                                                                 |
+| spring.datasource.password                | Database password                         | root                                             | ${db_password}                                                       |
```
Comment (PaulBredl) [inline README.md:17]:
```suggestion
| spring.datasource.password                | Database password                         | root                                             | *secret*                                                       |
```

**Review by PaulBredl — CHANGES_REQUESTED** [review]
`server.port` is still missing

File: README.md (hunk 1 above)
Comment (Keyshal-211) [reply README.md]: > This is not set in DEV

yes I already Removed this in my previous commit

File: README.md [hunk 4]
```diff
@@ -4,15 +4,42 @@
 
 The service is structured similar to the microservice template.
 
+## Environment variables
+
+## Relevant for deployment
```
Comment (PaulBredl) [inline README.md:9]:
```suggestion
### Relevant for deployment
```

File: README.md [hunk 5]
```diff
@@ -4,15 +4,42 @@
...
+|----------------------------|------------------------------------|--------------------------------------------------|----------------------------------------------------------------------|
+| spring.datasource.url      | PostgreSQL database URL            | jdbc:postgresql://localhost:4032/content-service | jdbc:postgresql://content-service-db-postgresql:5432/content-service |
+| spring.datasource.username | Database username                  | root                                             | gits                                                                 |
+| spring.datasource.password | Database password                  | root                                             | secret                                                               |
+| DAPR_HTTP_PORT             | Dapr HTTP Port                     | 4000                                             | 3500                                                                 |
+| server.port                | Port on which the application runs | 4001                                             | 4001                                                                 |
+
```
Comment (PaulBredl) [inline README.md:18]:
```suggestion

### Other properties
```

**Review by PaulBredl — APPROVED** [review]

**Pull request merged**